xun/tests/test_compatibility.py

```diff
@@ -0,0 +1,41 @@
+from xun.functions.compatibility import ast as compat_ast
+import ast
+
+
+def test_Constant():
+    constant_int = compat_ast.Constant(5)
+    constant_float = compat_ast.Constant(5.5)
+    constant_complex = compat_ast.Constant(5.5 + 5j)
+    constant_string = compat_ast.Constant("hello")
+    constant_bytes = compat_ast.Constant(b"0042")
+    constant_nameconstant_bool = compat_ast.Constant(True)
+    constant_nameconstant_none = compat_ast.Constant(None)
+    constant_ellipsis = compat_ast.Constant(...)
+
+    assert isinstance(constant_int, compat_ast.Constant)
+    assert isinstance(constant_float, compat_ast.Constant)
+    assert isinstance(constant_complex, compat_ast.Constant)
+    assert isinstance(constant_string, compat_ast.Constant)
+    assert isinstance(constant_bytes, compat_ast.Constant)
+    assert isinstance(constant_nameconstant_bool, compat_ast.Constant)
+    assert isinstance(constant_nameconstant_none, compat_ast.Constant)
+    assert isinstance(constant_ellipsis, compat_ast.Constant)
+
+    try:
+        # This should pass in Pythono 3.8, but fail in Python 3.6 and 3.7
+        # because compat_ast.Constant is made new then, but ast.Constant is
+        # still defined (no AttributeError).
+        assert compat_ast.Constant is ast.Constant
+    except AssertionError:
```

Comment:
This seems quite strange and should probably not be done. The reason it is done is that ast.Constant will be defined in Python versions 3.6, 3.7 and 3.8, but compat_ast.Constant is only equal to ast.Constant in Python version 3.8. Any suggestions for how to write a passing test for 3.6 and 3.7, @JensGM ?

Reply:
At this point, I'd say just add a check, note that in the test there is not point in defining the major and minor versions with an underscore. We only do it in ast.py because there the interface is very specific